theoplayer/how-to-guides/01-ads/12-mediatailor.md

```diff
@@ -96,6 +96,10 @@ val mediaTailorSource = MediaTailorSource
 theoplayerView.player.source = SourceDescription
     .Builder(mediaTailorSource)
     .build()
+    
+theoplayerView.player.source = SourceDescription
+    .Builder(mediaTailorSource)
+    .build() 
```

Comment:
FFS, now this thing is back. Gotta revert that again.

I really hope this is the last hotfix on 9.8.x...